hadoop-ozone/dist/src/main/compose/ozone-csi/docker-compose.yaml

```diff
@@ -24,7 +24,7 @@ services:
     env_file:
       - docker-config
     environment:
-      HADOOP_OPTS: ${HADOOP_OPTS}
+      OZONE_OPTS:
```

Comment:
For all these replacements, shouldn't we still provide ${OZONE_OPTS} to the environment as we did with HADOOP_OPTS before? We specify the coverage related options via this variable in test_all.sh for example.

Reply:
> Environment variables with only a key are resolved to their values on the machine Compose is running on

https://docs.docker.com/compose/compose-file/#environment

So repeating the variable name is unnecessary.